### PR TITLE
Submitting cerbos/cerbos

### PR DESCRIPTION
**Pull request by alexolivier**

### CODE REVIEW

**Comment by recena** [comment]
@alexolivier Did you read [this documentation](https://github.com/rancher/partner-charts?tab=readme-ov-file#suse-ready-partnership)?

**Comment by alexolivier** [comment]
Hi,

We are working through the partnership process already and sending this PR was part of that.

We have tested this chart on k3s and documentation is on github.com/cerbos/cerbos and docs.cerbos.dev for using the Helm chart. The license is Apache 2.0

File: packages/cerbos/cerbos/overlay/app-readme.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+##Cerbos
```
Comment (recena) [inline packages/cerbos/cerbos/overlay/app-readme.md:1]:
```suggestion
## Cerbos
```

**Review by recena — DISMISSED** [review]

**Comment by troytop** [comment]
Partner agreement processed. Welcome to the Rancher Partner Catalog Cerbos!

**Review by recena — APPROVED** [review]

**Review by adamkpickering — APPROVED** [review]

**Pull request merged**